JeMPI_Apps/JeMPI_Linker/src/main/java/org/jembi/jempi/linker/BackEnd.java

```diff
@@ -100,6 +109,19 @@ static void updateGoldenRecordField(
             final var result = libMPI.updateGoldenRecordField(goldenId, fieldName, maxEntry.getKey());
             if (!result) {
                LOGGER.error("libMPI.updateGoldenRecordField({}, {}, {})", goldenId, fieldName, maxEntry.getKey());
+            }  else {
+               // Update scores of all associated entities
+               mpiPatientList.forEach(mpiPatient -> {
+                  final var patient = mpiPatient.patientRecord();
+                  final var score = calcNormalizedScore(expandedGoldenRecord.goldenRecord().demographicData(), patient.demographicData());
+                  final var reCompute = libMPI.reComputeScores(patient.patientId(), goldenId, score);
+                  if (!reCompute) {
+                     LOGGER.error("Failed to update score for entity with UID {}", patient.patientId());
+                  } else {
+                     LOGGER.debug("Successfully updated score for entity with UID {}", patient.patientId());
+                  }
+               });
+               LOGGER.error("libMPI.updateGoldenRecordField({}, {}, {})", goldenId, fieldName, maxEntry.getKey());
```

Comment:
@mahaomolise92 this doesnt seem to be required. It will always execute for the recompute case

Reply:
we may not get a response from LibMPI, or maybe there is a problem with our query from LibMPI although this is highly unlikel;y but I just wanted to cater for that so that we avoid a crushing system